CoherentTasks/CoherentTasks/MaxMinArraySum.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoherentTasks
+{
+    class MaxMinArraySum
+    {
+        public static void Execute()
```

Comment:
Please read the task requirements carefully, at least "The application asks the user for the number of elements in a numeric array (at least two), and then in a loop -the elements themselves (of int type)".

Reply:
Sorry for this, lost my focus